pykeepass/pykeepass.py

```diff
@@ -560,3 +566,23 @@ def delete_binary(self, id):
         )
         for reference in binaries_gt:
             reference.id = reference.id - 1
+
+
+def create_database(filename, password=None, keyfile=None, transformed_key=None):
+    kdbx = KDBX.parse_file(
+        BLANK_DATABASE_LOCATION,
+        password=BLANK_DATABASE_PASSWORD,
+        keyfile=None,
+        transformed_key=None
+    )
+
+    with open(filename, 'wb') as f:
+        f.write(
+            KDBX.build(
+                kdbx,
+                password=password,
+                keyfile=keyfile,
+                transformed_key=transformed_key
+            )
+        )
```

Comment:
Why aren't you using the `read` and `save` functions here?

Reply:
Good point. Didn't really occur to me. I've updated the function to use the PyKeePass object instead to create the blank database.